### DISCUSSION TITLE
Detection if display connected ?

### DISCUSSION THREAD

**Discussion by herbert131** [olikraus/u8g2#2555] [Q&A]
Hi, how can I detect if a display is connected or not. begin delivers always true ? 
any Idea ?

**Comment by olikraus** [comment]
Difficult topic. Background: Older SPI display don't included any option to transfer data back to the SPI master. This means there are SPI displays, which can't be detected.
This has changed a little bit, but because of that reason, u8g2 doesn't include any read function from the display.

For I2C displays, you could check for the ACK response, so my suggestion is this: If  you need to be able to detect a display, use a display with I2C interface.
